application/core/navigation/library/build.gradle.kts

```diff
@@ -3,7 +3,4 @@ androidLibrary(
     dependencies = deps(
         androidx.navigation
     )
-)
-// TODO https://github.com/formatools/forma/issues/35
-// Need configurable plugin integration here
-apply(plugin = "androidx.navigation.safeargs.kotlin")
\ No newline at end of file
+).withPlugin(pluginWrappers.navigationSafeArgs)
```

Comment:
Should we establish a good practice by creating `androidLibraryWithNavigation(...) = androidLibrary(...).withPlugin(...)` wrapper function, so we'll make targets description in the same style?

Reply:
IMHO, creating new method for each combination of plugins is useless, it doesn't sole our tasks. We can use plugins for several targets? f.e. Kotlin.serialization can be used in library, utils, iml, androidLibrary, androidUtils and in this case we must create 5 copipaste of methods, if I understand you correctly. Also we can have combinations of plugins. In my opinion patterns like builder or decorator help us to combine several behaviors and keep it scalable without necessity to duplicate code

Reply:
Yea, in now days projects we'll get different combinations of custom plugins. In that case, many methods with name templates as `[target name]With[combination plugins] will be overhead for project self.